examples/custom_dimensions_example_test.go

```diff
@@ -1,4 +1,5 @@
-//+build examples
+//go:build examples
+// +build examples
```

Comment:
`//go:build examples` is the new directive and `// +build examples` is there for older versions of Go. Is it correct?

Reply:
Yes